.github/scripts/smoke_test.sh

```diff
@@ -0,0 +1,78 @@
+#! /bin/bash
+
+# Software test
+test_software() {
+cd ../../.. && \
+if [ -d "./$project" ]; then
+    rm -r $project
+fi
+dbt init -s $project && \
+{ cat > ./.dbt/profiles.yml << EOF
+$project:
+  outputs:
+    dev:
+      password: $test_password
+      port: 9047
+      software_host: $test_host
+      threads: 1
+      type: dremio
+      use_ssl: false
```

Comment:
Should we enable the option to have use_ssl be true?

Reply:
Depends on the cluster we plan to use. At the moment we're using one that doesn't have ssl enabled.

Reply:
Although the current cluster we are testing against doesn't have ssl enabled, ssl enabled clusters will definitely be tested against in the future, and I feel as though having an extra variable would be nice here.